sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added support for parsing the `expires_on` field returned by the `az` CLI in `AzureCliCredential`. ([#33947](https://github.com/Azure/azure-sdk-for-python/issues/33947))
```

Comment:
Added support for parsing the `expires_on` field returned by the new version of `az` CLI in `AzureCliCredential`.?

Reply:
Yea, I went ahead and made this a bit more explicit by specifying `az` versions >= 2.54.0.